site/content/docs/user-guide/setup/install.md

```diff
@@ -71,7 +71,7 @@ Krew self-hosts).
 
 1. Make sure `git` is installed on your system.
 1. Download `krew.exe` from the [Releases][releases] page to a directory.
-1. Launch a command-line window (`cmd.exe`) and navigate to that directory.
+1. Launch a command-line window (`cmd.exe`) with elevated privileges (to update symbolic link) and navigate to that directory.
```

Comment:
Thanks for this. I'm not sure if the `(...)` part is necessary.

If you think it is, maybe rewrite it as `(since the installation requires use of symbolic links)` ?

Reply:
sure, why not